cloud_node:
  ros__parameters:
    min_range: 0.15                 
    max_distance: 150.0                
    degree_mode: 2 
    distance_unit: 0.25
    return_mode: 1
    config_vert: true 
    print_vert: false  
    scan_frame_id: laser_link
    scan_num: 8
    publish_scan: false
    time_synchronization: false
    
lslidar_node:
  ros__parameters:
    device_ip: 192.168.1.200
    msop_port: 2368
    difop_port: 2369
    frame_id: laser_link       
    add_multicast: false
    group_ip: 224.1.1.2
    rpm: 600.0
    return_mode: 1
    degree_mode: 2        
    time_synchronization: false
    model: LSC16
    output_packets_topic: lslidar_packet
    output_difop_topic: lslidar_packet_difop
